<?xml version="1.0" encoding="UTF-8" ?>
<launch>
	<arg name="port" default="/dev/ttyUSB1"/>
	<arg name="output_topic" default="image_resize"/>

	<node pkg="xbee_communication" type="xbee_image_receiver.py" name="xbee_image_receiver"  output="screen">
    <param name="port" value="$(arg port)" />
    <param name="output_topic" value="$(arg output_topic)" />
	</node>

</launch>
